### DISCUSSION TITLE
ِDynamic router title

### DISCUSSION THREAD

**Discussion by botify-framework** [laravel/framework#33239] [General] [answered]
Hi,
Please add an attribute for the custom title for the routes.

```
$router->get('/', [
    'as'=> 'home',
    'uses'=> 'HomeController@index',
    'title'=> 'Home page',
]);

```

And in blade:

`{{ Route::getTitle() }}
`

Thanks.

**Comment by browner12** [comment, marked as answer]
This is not the responsibility of the router.

**Reply by botify-framework** [reply]
But it can also be a great help in determining the title of pages dynamically.
Without using direct and writing additional codes.

**Reply by botify-framework** [reply]
Instead of commenting, you and those who agree with you should use additional code and @section and @yield directives to dynamize the title pages.
